packages/lab/src/__tests__/__e2e__/dialog/dialog.cy.tsx

```diff
@@ -0,0 +1,154 @@
+import { composeStories } from "@storybook/react";
+import * as dialogStories from "@stories/dialog/dialog.stories";
+
+const composedStories = composeStories(dialogStories);
+
+const { Default, Accent, InfoStatus } = composedStories;
+
+describe("GIVEN a Dialog", () => {
+  describe("WHEN no props are provided", () => {
+    it("THEN it should display an overlay by default", () => {
+      cy.mount(<Default />);
+
+      cy.findByRole("button", { name: /Open Dialog/i }).click();
+
+      cy.get(".saltDialog").should("be.visible");
+      cy.get(".saltDialogTitle").should("be.visible");
+      cy.get(".saltDialogContent").should("be.visible");
+      cy.get(".saltDialogActions").should("be.visible");
+    });
+
+    it("THEN it should display animations by default", () => {
+      cy.mount(<Default />);
+
+      cy.findByRole("button", { name: /Open Dialog/i }).click();
+
+      cy.get(".saltDialog").should("have.class", "saltDialog-enterAnimation");
+    });
+
+    it(
+      "THEN it should display medium size by default",
+      {
+        viewportHeight: 900,
+        viewportWidth: 1921,
+      },
+      () => {
+        cy.mount(<Default />);
+
+        cy.findByRole("button", { name: /Open Dialog/i }).click();
+
+        cy.get(".saltDialog").should("have.class", "saltDialog-medium-xl");
+      }
+    );
+  });
+
+  describe("WHEN a size is provided", () => {
```

Comment:
This only tests the class is applied and not the actual visuals. Is it possible to use a chromatic test here?

Reply:
yup - will look into the chromatic tests